frames/src/main/java/com/checkout/frames/screen/billingaddress/billingaddressform/BillingAddressFormScreen.kt

```diff
@@ -0,0 +1,65 @@
+package com.checkout.frames.screen.billingaddress.billingaddressform
+
+import androidx.compose.animation.AnimatedContentScope
+import androidx.compose.animation.ExperimentalAnimationApi
+import androidx.compose.animation.core.tween
+import androidx.compose.runtime.Composable
+import androidx.lifecycle.viewmodel.compose.viewModel
+import com.checkout.frames.di.base.Injector
+import com.checkout.frames.screen.billingaddress.billingaddressdetails.BillingAddressDetailsScreen
+import com.checkout.frames.screen.countrypicker.CountryPickerScreen
+import com.checkout.frames.screen.navigation.Screen
+import com.checkout.frames.style.screen.BillingFormStyle
+import com.google.accompanist.navigation.animation.AnimatedNavHost
+import com.google.accompanist.navigation.animation.composable
+import com.google.accompanist.navigation.animation.rememberAnimatedNavController
+
+@SuppressWarnings("MagicNumber")
+@OptIn(ExperimentalAnimationApi::class)
+@Composable
+internal fun BillingAddressFormScreen(
+    style: BillingFormStyle,
+    injector: Injector,
+    onClose: () -> Unit
+) {
+    val animationDuration = 350
+    val navController = rememberAnimatedNavController()
```

Comment:
To avoid misleadings in the future, I would suggest renaming it to childNavController.